google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITSmokeInstancesTest.java

```diff
@@ -213,20 +201,19 @@ public void testDefaultClient() throws IOException {
             .addDisks(DISK)
             .addNetworkInterfaces(NETWORK_INTERFACE)
             .build();
-    Operation operation = defaultClient.insert(DEFAULT_PROJECT, DEFAULT_ZONE, instanceResource);
-    waitUntilStatusChangeTo(operation);
+    defaultClient.insertAsync(DEFAULT_PROJECT, DEFAULT_ZONE, instanceResource).get();
     instances.add(instanceResource);
     assertInstanceDetails(getInstance());
   }
 
   @Test
-  public void testDefaultResource() {
+  public void testDefaultResource() throws ExecutionException, InterruptedException {
     Instance instanceResource = Instance.newBuilder().build();
     try {
-      instancesClient.insert(DEFAULT_PROJECT, DEFAULT_ZONE, instanceResource);
+      instancesClient.insertAsync(DEFAULT_PROJECT, DEFAULT_ZONE, instanceResource).get();
       fail("Did not catch the exception");
-    } catch (InvalidArgumentException ex) {
-      String message = "Bad Request";
+    } catch (ExecutionException ex) {
```

Comment:
ExecutionException is caught here, but is also declared in throws block above. Do you really need both?

Reply:
We don't, removed from declaration